packages/ibm-products/scripts/generate/templates/DISPLAY_NAME.tsx

```diff
@@ -52,7 +60,7 @@ export let DISPLAY_NAME = React.forwardRef(
 
       // Collect any other property values passed in.
       ...rest
-    },
+    }: DISPLAY_NAMEProps,
     ref
```

Comment:
Adding the following will get rid of a ts error from the forwarded ref

```
ref: ForwardedRef<HTMLDivElement>
```

![image](https://github.com/user-attachments/assets/033ad40f-d1d0-4b87-8c95-add44e4e462f)